docs/IconField.md

```diff
@@ -45,7 +45,7 @@ It will parse children based on the `isField` and `isIcon` static properties.
 By default the design system `Input`, `Select`, and `Icon` components will work with the `IconField` component, but to allow other children to render, add an appropriate static property.
 
 ```jsx
-import React from 'react'
+import * as React from 'react'
```

Comment:
Do we need the changed import syntax since we're not turning on TS plugins?

Reply:
maybe not? I thought I remembered reading that this was the new best practice..is that only for TS?